packages/auth/src/access-tokens.test.ts

```
@@ -0,0 +1,546 @@
+    await test.cleanup();
+  });
+
+  it("returns bearer credentials without setting auth cookies", async () => {
+    const response = await fragments.auth.callRoute("POST", "/sign-up", {
+      body: { email: "bearer-only@test.com", password: "password123" },
+    });
+
+    assert(response.type === "json");
+    assert(response.data.auth.kind === "jwt");
+    expect(response.data.auth.refreshToken).toEqual(expect.any(String));
+    assert(response.headers.get("Set-Cookie") ?? "" === "");
```

Biome (2.4.16)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)